pecan/STATUS

```diff
@@ -0,0 +1 @@
+META	2015-03-13 17:07:21
```

Comment:
pecan/STATUS should not be committed. Previous revisions had finally removed the pecan/pecan directory and it doesn't need to be reintroduced. Please remove both file and folder and then add the whole folder to your .gitignore